### ISSUE TITLE
Only allows single FX imports

### ISSUE THREAD

**Issue by ekinnee** [catman85/Bias-Importer#3]
Unless I'm doing it wrong, it seems you can only import one Bias FX preset at at time. Are the command line arguments to designate the path and preset file?

**Comment by catman85** [comment]
Yes you are right. This app only allows you to import one preset at a time. 
There is not a command line implementation of the app only a graphical one. 
Inside the UI you need to designate the path and preset file.

Please Note that moving your entire preset library is super easy, you just need to copy and paste a folder.

At the time I am writing this, importing multiple presets requires you to use the app multiple times.
